From 5a4d9800c076d5e590fb2b6f8ec98d9565fd2189 Mon Sep 17 00:00:00 2001
From: Henrik <henrik@affekt.org>
Date: Mon, 31 Jul 2017 13:18:56 +0200
Subject: [PATCH] remove colon from token delimiter

this is causing problems for people using colons in their password
---
 config.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config.cpp b/config.cpp
index 38e10ff..15394f6 100644
--- a/config.cpp
+++ b/config.cpp
@@ -10,10 +10,10 @@ void CConfig::ParseLine(std::string line)
 
 	strncpy(szline, line.c_str(), line.size()+1);
 
-	pstr = strtok(szline, " :=\t");
+	pstr = strtok(szline, " =\t");
 	while(pstr) {
 		tokens.push_back(pstr);
-		pstr = strtok(NULL, " :=\t");
+		pstr = strtok(NULL, " =\t");
 	}
 	delete[] szline;
 
-- 
2.13.3

